packages/grafana-ui/src/components/TagsInput/TagsInput.tsx

```diff
@@ -73,6 +73,7 @@ export const TagsInput = ({
       <Input
         id={id}
         disabled={disabled}
+        readOnly={disabled}
```

Comment:
As previous comment can't we continue to use `disabled` here?

Reply:
fixed